<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ex2</class>
 <widget class="QWidget" name="Ex2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ex2</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lAttName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="WSlider" name="hslWriter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedControls">
      <bool>true</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QDoubleSpinBox" name="sbValue"/>
   </item>
   <item row="1" column="3">
    <widget class="TPushButton" name="tpbApply">
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="ReadLabel" name="qlRead">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TPushButton</class>
   <extends>QPushButton</extends>
   <header>TPushButton</header>
  </customwidget>
  <customwidget>
   <class>ReadLabel</class>
   <extends>QLabel</extends>
   <header>readlabelb.h</header>
  </customwidget>
  <customwidget>
   <class>WSlider</class>
   <extends>QSlider</extends>
   <header>wslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
